xpcPlugin/MessageHandlers.cpp

```diff
@@ -573,7 +573,7 @@ namespace XPC
 
 		if (aircraftNumber > 0)
 		{
-			// Enable AI for the aircraftNumber we are setting
+			// Enable AI for the aircraftNumber we are setting: MRT COMMENT: ENABLE, OR DISABLE?
```

Comment:
Enable is correct. Setting the `override_planepath` dref disables AI ([see this page](http://www.xsquawkbox.net/xpsdk/mediawiki/MovingThePlane#Controlling_AI_Aircraft_in_X-Plane_8_and_9)). This is here to re-enable the AI after setting the position. It occurs to me that this could conflict if the user is manually disabling the AI, but that isn't a use-case that we have been asked to support yet.